web/src/main/java/org/perfrepo/web/adapter/AlertAdapter.java

```diff
@@ -0,0 +1,61 @@
+package org.perfrepo.web.adapter;
+
+import org.perfrepo.dto.alert.AlertDto;
+
+import java.util.List;
+
+/**
+ * Service adapter for {@link AlertDto} object. Adapter supports CRUD and another
+ * operations over this object.
+ *
+ * @author Jiri Grunwald (grunwjir@gmail.com)
+ */
+public interface AlertAdapter {
+
+    /**
+     * Return {@link AlertDto} test alert by its id.
+     *
+     * @param id The alert identifier.
+     * @return Found {@link AlertDto} test alert.
+     * @throws org.perfrepo.web.adapter.exceptions.NotFoundException If the test alert does not exist.
+     * @throws org.perfrepo.web.adapter.exceptions.AdapterException If anything bad happened.
+     */
+    AlertDto getAlertById(Long id);
```

Comment:
Let's name it just getAlert(Long id), it's clear from the method signature that it's by id :)